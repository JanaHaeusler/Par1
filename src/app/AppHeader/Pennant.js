import styled from 'styled-components/macro'
import { LogoIcon } from '../assets/Icons/Icons'

export default function Pennant() {

  return (
      <PennantStyled>
        <LogoIconStyled/>
      </PennantStyled>
  )
}

const PennantStyled = styled.div`
  display: flex; 
  justify-content: center;
  width: 60px;
  height: 60px;
  box-shadow: 0 0 10px var(--primary-dark);
  border-radius: 0 0 50px 50px;
  background-color: var(--text-light);
`
const LogoIconStyled = styled(LogoIcon)`
  margin-top: 10px;
  width: 30px;
  fill: var(--secondary-dark);
`