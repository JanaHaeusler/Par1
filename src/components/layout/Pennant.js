import styled from 'styled-components/macro'
import { Logo } from '../Icons'

export default function Pennant() {

  return (
      <PennantStyled>
        <LogoStyled/>
      </PennantStyled>
  )
}

const PennantStyled = styled.div`
  display: flex; 
  justify-content: center;
  width: 60px;
  height: 60px;
  box-shadow: 0 0 10px var(--primary-dark);
  border-radius: 0 0 50px 50px;
  background-color: var(--text-light);
`
const LogoStyled = styled(Logo)`
  margin-top: 10px;
  width: 30px;
  fill: var(--secondary-dark);
`