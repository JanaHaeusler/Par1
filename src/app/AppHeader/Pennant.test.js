import Pennant from './Pennant'
import { render } from '@testing-library/react'

describe('Pennant', () => {
    it('renders correctly', () => {
        const { container } = render(<Pennant/>)
        expect(container.firstChild).toMatchSnapshot()
    })
})