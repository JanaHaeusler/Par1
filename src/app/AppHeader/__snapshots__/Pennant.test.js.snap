// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pennant renders correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  width: 60px;
  height: 60px;
  box-shadow: 0 0 5px var(--secondary-medium);
  border-radius: 0 0 50px 50px;
  background-color: var(--light);
}

.c1 {
  margin-top: 10px;
  width: 30px;
  fill: var(--primary);
}

<div
  class="c0"
>
  <svg
    class="c1"
  >
    golf-logo.svg
  </svg>
</div>
`;
